.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
```

Comment:
@jasonacox / @mcbirse I'm just assuming 2 spaces vs tabs is your preference.

Not every file matches this setting.

The editorconfig does NOT force changes, but does attempt to tell the editor how it should behave when you're adding new code.
Please make sure this is your preference before squashing

Reply:
We have a mix, but 4 spaces is my preference (as seen in most of the *py, *sh and *yml) but I know we have 2 spaces in *conf.